src/components/Navbar/Navbar.module.scss

```diff
@@ -137,17 +138,112 @@
   .desktopMenuDark {
     color: $white;
   }
-  .desktopMenuItem {
-    margin-right: 26px;
+  .desktopMenuItems {
     font-size: 16px;
     margin-top: 50px;
     height: 24px;
     display: flex;
     align-items: center;
+    div * {
+      margin-right: 26px;
+    }
+    div {
+      display: inline-block;
+      position: relative;
+    }
+    div ::before {
+      content: "";
+      position: absolute;
+      bottom: 4.3px;
+      width: 0;
+      height: 1.2px;
+      background-color: $black;
+      transition: width 0.25s ease-out;
+    }
+  }
+  .desktopMenuItemsDark {
```

Comment:
panie tak to to ja tego nie wciągnę, nie moze byc ustawiane z palca, musi być dynamiczne od wielkosci tekstu w środku a co jak ktoś ma inny ekran i u niego tekst ma inny rozmiar? wtedy bedzie inny width
weź tak zrób jak chłop i będzie na 100% lepiej 
https://www.youtube.com/watch?v=aswRKAjjWuE